dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -375,7 +387,13 @@ class ApplicationsControllerTest < ::ActionController::TestCase
 
     test 'update sends a decision email once if status changed to decision and if associated with an RP' do
       sign_in @program_manager
-      Pd::Application::TeacherApplicationMailer.expects(:accepted).once
+      summer_workshop = create :summer_workshop
+      @csd_teacher_application_with_partner.update(pd_workshop_id: summer_workshop.id)
+      @csd_teacher_application_with_partner.reload
```

Comment:
I'm worried about this interfering with the regional partner tests that count the number of workshops there are? But if it's not failing drone, then ignore me!

Reply:
It's not interfering since the new workshop and application updates are all being applied within that specific test!

Reply:
But it's the same `@csd_teacher_application_with_partner` instance variable, so they are pointing to the same object in memory?

Reply:
True, but I thought with the setup it would be re-defining the variables with each test right?